maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -149,7 +157,18 @@ def submit(self, step, path, cwd, job_map=None, env=None):
         :returns: The return status of the submission command and job
         identiifer.
         """
-        cmd = " ".join(["sbatch", path, "-D", cwd])
+        # Check and see if we should be submitting into a reservation.
+        if self._batch["reservation"]:
```

Comment:
style: I feel like this could be cleaned up. Build a collection for command? Something like

```
args = ["sbatch"]

if self._batch["reservation"]:
    args += ["--reservation", self._batch["reservation"]]

args += ["-D", cwd, path]
cmd = " ".join(args)
```